apps/new-tab/src/App.tsx

```
@@ -656,22 +669,29 @@ const App: Component = () => {
                   layout() == "top" && "!pl-8"
                 )}
+                id="widgets"
               >
                 {filteredWidgets().length > 0 ? (
-                  filteredWidgets().map((item: any) => {
+                  filteredWidgets().map((item: any, index: number) => {
                     return (
                       <div
                         class={`${uuidv4()} slot group h-fit`}
                         data-swapy-slot={item}
                       >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)